spec/merch_calendar/retail_calendar_spec.rb

```diff
@@ -121,5 +152,84 @@
       expect(merch_months[10].strftime('%Y-%m-%d')).to eq '2014-11-30'
     end
   end
-
+  
+  describe "#merch_year_from_date" do
+    it "returns the correct merch calendar year" do
+      expect(subject.merch_year_from_date(Date.new(2012, 1, 28))).to eq 2011
+      expect(subject.merch_year_from_date(Date.new(2012, 1, 29))).to eq 2012
+      expect(subject.merch_year_from_date(Date.new(2018, 1, 24))).to eq 2017
+      expect(subject.merch_year_from_date(Date.new(2018, 2, 3))).to eq 2017
+      expect(subject.merch_year_from_date(Date.new(2018, 2, 4))).to eq 2018
+      expect(subject.merch_year_from_date(Date.new(2019, 2, 2))).to eq 2018
+      expect(subject.merch_year_from_date(Date.new(2019, 2, 3))).to eq 2019
+    end
+  end
+  
+  describe "#merch_to_julian" do
+    it "converts merch months to julian months" do
+      expect(subject.merch_to_julian(1)).to eq 2
+      expect(subject.merch_to_julian(2)).to eq 3
+      expect(subject.merch_to_julian(3)).to eq 4
+      expect(subject.merch_to_julian(4)).to eq 5
+      expect(subject.merch_to_julian(5)).to eq 6
+      expect(subject.merch_to_julian(6)).to eq 7
+      expect(subject.merch_to_julian(7)).to eq 8
+      expect(subject.merch_to_julian(8)).to eq 9
+      expect(subject.merch_to_julian(9)).to eq 10
+      expect(subject.merch_to_julian(10)).to eq 11
+      expect(subject.merch_to_julian(11)).to eq 12
+      expect(subject.merch_to_julian(12)).to eq 1
+      expect { subject.merch_to_julian(13) }.to raise_error ArgumentError
+      expect { subject.merch_to_julian(0) }.to raise_error ArgumentError
+    end
+  end
+  
+  describe "#julian_to_merch" do
+    it "converts julian months to merch months" do
+      expect(subject.julian_to_merch(2)).to eq 1
+      expect(subject.julian_to_merch(3)).to eq 2
+      expect(subject.julian_to_merch(4)).to eq 3
+      expect(subject.julian_to_merch(5)).to eq 4
+      expect(subject.julian_to_merch(6)).to eq 5
+      expect(subject.julian_to_merch(7)).to eq 6
+      expect(subject.julian_to_merch(8)).to eq 7
+      expect(subject.julian_to_merch(9)).to eq 8
+      expect(subject.julian_to_merch(10)).to eq 9
+      expect(subject.julian_to_merch(11)).to eq 10
+      expect(subject.julian_to_merch(12)).to eq 11
+      expect(subject.julian_to_merch(1)).to eq 12
+      expect { subject.julian_to_merch(13) }.to raise_error ArgumentError
+      expect { subject.julian_to_merch(0) }.to raise_error ArgumentError
+    end
+  end
+  
+  describe "#weeks_for_month" do
+    context "correct number of weeks given julian month and fiscal year" do
```

Comment:
I don't think this qualifies as a `context`.   Contexts are used to group related specs together, usually indicating a precondition of all specs that fall under it.

ex...

```
describe "a login page" do
  context "user is already logged in" do
    it "tells the user they're already logging in" do
...
    it "takes the user to the 'logged in user' page" do
...
```

Reply:
Decided to delete the context block